src/setlist-fm-helper.ts

```diff
@@ -3,6 +3,10 @@ import {instrumentCall} from "./helpers";
 const baseUrl = "https://api.setlist.fm/rest/1.0/";
 
 async function getArtistSetlists(artistMbid: string): Promise<SetlistFmSetlist[]> {
+
+    if (!process.env.SETLIST_FM_API_KEY || process.env.SETLIST_FM_API_KEY === "")
```

Comment:
I want this to error on prod if something is wrong with the API key. Will you wrap these two checks in () and add a DEPLOY_STAGE !== PROD (double check me on the correct values I'm going off the top of my head) check as the first clause of the if statement please?

Reply:
Fixed